src/order/order-action-creator.spec.js

```diff
@@ -224,6 +224,8 @@ describe('OrderActionCreator', () => {
 
             expect(actions).toEqual([
                 { type: OrderActionType.SubmitOrderRequested },
+                { type: OrderActionType.LoadOrderRequested },
```

Comment:
I'm wondering whether we should have actions and subactions to differentiate what part of the process failed. If we created the order successfully but failed to hydrate it from storefront API, should we aim to create the order again? or just call again storefront API?

Something similar happens with sign in user. If a user signed in successfully but failed to load Checkout, will subsequent logins work? Or should we just reattempt to load checkout?

Reply:
So at the moment, we don't automatically recover. The user can trigger the action again themselves.